### PR TITLE
Implementation

### PR DESCRIPTION
**Pull request by mihaidma**

### CODE REVIEW

**Comment by mcdonnelldean** [comment]
@mihaidma very nice! LGTM

File: srv/start.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+'use strict'
+
+var Seneca = require('seneca')
```
Comment (mirceaalexandru) [inline srv/start.js]: Use please ES6

File: srv/start.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    log: 'none'
+  },
+  coveralls: {
+    registry: envs.NPM_REGISTRY || 'http://registry.npmjs.org/',
+    url: envs.COVERALLS_REGISTRY || 'https://coveralls.io/'
+  },
+  mesh: {
+    auto: true,
+    listen: [
+      {pin: 'role:coveralls,cmd:get', model: 'consume'},
```
Comment (mirceaalexandru) [inline srv/start.js]: each pin should have defined also host.

File: srv/start.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    tag: envs.COVERALLS_TAG || 'nodezoo-coveralls',
+    log: 'none'
+  },
+  coveralls: {
+    registry: envs.NPM_REGISTRY || 'http://registry.npmjs.org/',
+    url: envs.COVERALLS_REGISTRY || 'https://coveralls.io/'
+  },
+  mesh: {
+    auto: true,
+    listen: [
```
Comment (mirceaalexandru) [inline srv/start.js]: We need here host & bases options

File: srv/start.js [hunk 4]
```diff
@@ -18,6 +18,8 @@ var opts = {
   },
   mesh: {
     auto: true,
+    host: envs.WEB_HOST || '127.0.0.1',
+    bases: [envs.BASE_HOST || '127.0.0.1:39999'],
     listen: [
       {pin: 'role:coveralls,cmd:get', model: 'consume'},
```
Comment (mirceaalexandru) [inline srv/start.js]: Hey @mihaidma you must add the host also on each pin.

So you will have something like this:

```
{pin: 'role:coveralls,cmd:get', model: 'consume', host: envs.WEB_HOST || '127.0.0.1'}
```

Also this is not WEB_HOST, is COVERALLS_HOST. Each instance has its own host env, do if all services are started on the same machine we can control the variables for each instance.

Only the BASE_HOST is common to all services.

**Pull request merged**